src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataInfo.cs

```diff
@@ -32,6 +34,18 @@ public ViewDataInfo(object container, PropertyInfo propertyInfo, Func<object> va
             _valueAccessor = valueAccessor;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ViewDataInfo"/> class with info about a
+        /// <see cref="ViewDataDictionary"/> lookup which is evaluated when <see cref="Value"/> is read. 
+        /// It uses <see cref="PropertyInfo.GetValue(object)"/> on <paramref name="propertyInfo"/> passing parameter <paramref name="container"/>.
+        /// </summary>
+        public ViewDataInfo(PropertyInfo propertyInfo, object container)
```

Comment:
- Not sure why the other constructors lack `<param>` doc comments but let's not make that gap worse. Please add for this new constructor.
- minor: Might be nice to stick this between the existing two constructors.
- Suggest keeping the parameter order consistent w/ existing constructor overloads. Then all would begin w/ the `container`.
- minor: Why not?
``` c#
public ViewDataInfo(object container, PropertyInfo propertyInfo)
    : this(container, propertyInfo, _propertyInfoResolver)
{
}
```